tooling-core/src/main/java/org/opentest4j/reporting/tooling/core/htmlreport/CoreContributor.java

```diff
@@ -79,6 +80,8 @@ public class CoreContributor implements Contributor {
 	private static final Pattern CAMEL_CASE_WORD_BOUNDARY_PATTERN = Pattern.compile("([a-z])([A-Z])");
 	private static final Set<String> SUPPORTED_IMAGE_MEDIA_TYPES = Set.of("image/png", "image/jpeg", "image/gif",
 		"image/avif", "image/webp", "image/svg+xml");
+	private static final Set<String> SUPPORTED_VIDEO_MEDIA_TYPES = Set.of("video/ogg", "video/mp4", "video/quicktime",
```

Comment:
From MDN I'm getting that the supported types are browser dependent. And that you can use a fall back element to link directly to the video when it not supported. So I would assume any media type that matches `^video/.*$` could be displayed as a video element.

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/video#usage_notes